### PR TITLE
Nexus 3 support

### PR DESCRIPTION
**Pull request by albustax**
This PR allows Nexus 3 installation and Debian 8 support for nexus service.

### CODE REVIEW

**Comment by pschichtel** [comment]
Great, Jessie support. My upgrade on Jessie just had problems because the init script messed up.

File: manifests/service.pp [hunk 1]
```diff
@@ -36,45 +36,69 @@
...
-    require => [File['/etc/init.d/nexus'],
-                File_line['nexus_NEXUS_HOME'],
-                File_line['nexus_RUN_AS_USER'],]
+      service{ 'nexus':
+        ensure  => running,
+        enable  => true,
+        status  => $status_line,
+        require => [File['/etc/init.d/nexus'],
+          File_line['nexus_NEXUS_HOME'],
+          File_line['nexus_RUN_AS_USER'],]
```
Comment (kenbreeman) [inline manifests/service.pp]: nit: remove the extra `,` and put `]` on a newline to match the styling above

File: templates/nexus.systemd.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[Service]
+Type=forking
+ExecStart=<%= scope.lookupvar('nexus::service::nexus_script') %> start
+ExecStop=<%= scope.lookupvar('nexus::service::nexus_script') %> stop
+User=<%= scope.lookupvar('nexus::service::nexus_user') %>
+Group=<%= scope.lookupvar('nexus::service::nexus_group') %>
+Restart=on-abort
+
+[Install]
+WantedBy=multi-user.target
```
Comment (kenbreeman) [inline templates/nexus.systemd.erb]: nit: newline

**Comment by kenbreeman** [comment]
This looks good to me but I don't have a Debian-based system handy to test on, how well tested is this?

**Comment by pranav** [comment]
lgtm too

**Comment by kenbreeman** [comment]
Published to forge: https://forge.puppet.com/hubspot/nexus/readme

**Comment by kenbreeman** [comment]
`1.4.0`

**Pull request merged**